htdocs/comm/action/peruser.php

```diff
@@ -645,19 +647,8 @@
 if ($status == 'todo') {
 	$sql .= " AND (a.percent >= 0 AND a.percent < 100)";
 }
-// We must filter on assignement table
-if ($filtert > 0 || $usergroup > 0) {
-	$sql .= " AND (";
-	if ($filtert > 0) {
-		$sql .= "ar.fk_element = ".$filtert;
```

Comment:
Filter on user seems to have been lost.

Reply:
No because in new version there is a line 554
if ($filtert > 0) {
		$sql .= " AND ar.fk_element = ".((int) $filtert);
}

on the INNER JOIN directly no need in the WHERE clause (better for use index)